rom/dev/src/rom.ld

```diff
@@ -20,9 +20,9 @@ ROM_ORG          = 0x00000000;
 ICCM_ORG         = 0x40000000;
 DCCM_ORG         = 0x50000000;
 DATA_ORG         = 0x50000000;
-STACK_ORG        = 0x5001C000;
-ESTACK_ORG       = 0x5001F800;
-NSTACK_ORG       = 0x5001FC00;
+STACK_ORG        = 0x5003C000;
```

Comment:
any comments to help with these allocations?

Reply:
**Previous Configuration:**
STACK_ORG was located at 0x5001C000.
ROM stack size was 14 KB (STACK_SIZE = 14K).
ESTACK and NSTACK were each allocated 1 KB.
The end of NSTACK was at 0x1FFFF, marking the end of the older DCCM.

**New Configuration:**
With the DCCM increasing by 128 KB, all these values are shifted downwards by 0x20000.